### PR TITLE
Kxz blog

### PR DESCRIPTION
**Pull request by kxz18**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
---
> Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by kxz18** [comment]
I forgot to update from upstream, which caused the conflicts. I didn't edit the two files which brought up conflicts, so just keep the master branch version of these two files.

**Comment by lijiarui** [comment]
Since we change the template or wechaty blog, we have a break change:

## Before
You should add your profile at `author.yml`

## After

Remove `author.yml`

### 1. Create a file at `_developers` named as `yourname.md` 

 `yourname` should be the same as the author name you put in the blog:
```
---
title: XX
date: XX
author: yourname
categories: XX
tags:
  - XX
image: XX
---
```
### 2. The file should begin with:
```
---
name: Jiarui LI (李佳芮)
site: https://lijiarui.github.io
bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
avatar: /assets/developers/lijiarui/avatar.png
email: rui@juzi.bot
twitter: https://weibo.com/u/2175505900
---
```

### 3. Example:

- lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
- It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

Looking forward to your profile file at the `_developers` folder!

File: _developers/kxz18.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 name: kxz
-site: https://github.com/kxz18
+site: https://kxz18.github.io/
 avatar: /assets/developers/kxz18/avatar.png
```
Comment (lijiarui) [inline _developers/kxz18.md:4]: Where is this file?

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+
+ title: "暑期2020 [基于python-wechaty的群聊助手机器人] POC 成果展示"
+ date: 2020-08-14 21:30 +0800
+ author: kxz18
+ categories: project
```
Comment (lijiarui) [inline _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md:6]: Please add a soc tag here. e.g.:
```
tags:
  - soc
```

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ header:
+    teaser: /assets/2020/groupchat-assistant-bot/header.jpeg
```
Comment (lijiarui) [inline _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md]: Where is this file?

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _developers/kxz18.md (hunk 1 above)
Comment (kxz18) [reply _developers/kxz18.md:4]: I found It's already in /assets/developers/kxz18/avatar.png when I pulled from upstream.

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md (hunk 2 above)
Comment (kxz18) [reply _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md:6]: In the new commit I have add the tags.

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md (hunk 3 above)
Comment (kxz18) [reply _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md]: This file was uploaded in last pull request (plan blog).

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
+---
+
+ title: "暑期2020 [基于python-wechaty的群聊助手机器人] POC 成果展示"
+ date: 2020-08-14 21:30 +0800
+ author: kxz18
+ categories: project
+ image: /assets/2020/groupchat-assistant-bot/header.jpeg
```
Comment (lijiarui) [inline _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md]: Well, I think you should create a new file for this blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md (hunk 4 above)
Comment (kxz18) [reply _posts/2020-08-14-python-wechaty-groupchat-assistant-bot-poc.md]: New file has been uploaded in latest commit.

**Review by lijiarui — APPROVED** [review]
Good job!

**Comment by lijiarui** [comment]
@kxz18 Please make ci green first.

**Review by lijiarui — APPROVED** [review]
Good job！

**Pull request merged**